.github/workflows/rails-tests-linters.yaml

```
@@ -37,6 +37,7 @@ jobs:
       RAILS_ENV: test
       TEST_DATABASE_URL: "postgres:///${{inputs.test_database_name || github.event.repository.name}}"
       TEST_CORE_ADMIN_DATABASE_URL: "postgres:///${{inputs.core_admin_test_database_name}}"
+      timeout-minutes: ${{ vars.RAILS_TEST_TIMEOUT_MINUTES }}
```

The option is on the same level as `runs-on` and `env`
```suggestion
    timeout-minutes: ${{ vars.RAILS_TEST_TIMEOUT_MINUTES }}
```